src/main/java/heronarts/lx/dmx/DmxRangeModulator.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2023- Justin K. Belcher, Mark C. Slee, Heron Arts LLC
+ *
+ * This file is part of the LX Studio software library. By using
+ * LX, you agree to the terms of the LX Studio Software License
+ * and Distribution Agreement, available at: http://lx.studio/license
+ *
+ * Please note that the LX license is not open-source. The license
+ * allows for free, non-commercial use.
+ *
+ * HERON ARTS MAKES NO WARRANTY, EXPRESS, IMPLIED, STATUTORY, OR
+ * OTHERWISE, AND SPECIFICALLY DISCLAIMS ANY WARRANTY OF
+ * MERCHANTABILITY, NON-INFRINGEMENT, OR FITNESS FOR A PARTICULAR
+ * PURPOSE, WITH RESPECT TO THE SOFTWARE.
+ *
+ * @author Mark C. Slee <mark@heronarts.com>
+ * @author Justin K. Belcher <justin@jkb.studio>
+ */
+
+package heronarts.lx.dmx;
+
+import heronarts.lx.LXCategory;
+import heronarts.lx.modulator.LXModulator;
+import heronarts.lx.parameter.BooleanParameter;
+import heronarts.lx.parameter.DiscreteParameter;
+import heronarts.lx.parameter.LXParameter;
+
+/**
+ * Modulator receiving a range from [min] to [max] within a DMX channel.
+ */
+@LXModulator.Global("DMX Range")
+@LXModulator.Device("DMX Range")
+@LXCategory(LXCategory.DMX)
```

Comment:
Not sure that Range is an intuitive name for this. Something like Conditional or Trigger maybe seems more appropriate, since that's what it is outputting.

Reply:
Also `implements LXTriggerSource`

Reply:
> Also implements LXTriggerSource

Yes!  Thanks.

Reply:
> Not sure that Range is an intuitive name for this. Something like Conditional or Trigger maybe seems more appropriate, since that's what it is outputting.

My primary intention was to map the range of input values to a normalized output.  The boolean out is helpful too, but the names Conditional or Trigger might hide the normalized function.  Other ideas?

This is common with DMX fixtures, to divide one channel into multiple alternative ranges.  Example:
<img width="409" alt="image" src="https://github.com/heronarts/LX/assets/6582491/9c063aa6-4250-41bb-a09f-ad524d38276c">

And an example where some ranges within a DMX channel are boolean and some are normalized:
<img width="404" alt="image" src="https://github.com/heronarts/LX/assets/6582491/8e5eb381-69e9-41eb-89f5-4eb5d0ccb9cc">
...In the case of 64-95 for strobe effect slow to fast, for example I would map the boolean `out` of this modulator to a LX `StrobeEffect.enabled` parameter, and map the normalized range to the `StrobeEffect.speed` parameter.